<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScanStep</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGraphicsView" name="leftView">
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="interactive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="leftLoadButton">
          <property name="text">
           <string>Загрузить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="leftMarkupButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Разметить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="leftParametersButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Характеристики</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="rightView">
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="interactive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="rightLoadButton">
          <property name="text">
           <string>Загрузить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rightMarkupButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Разметить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rightParametersButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Характеристики</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionSaveLeft"/>
    <addaction name="actionSaveRight"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewProject">
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveLeft">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить левое изображение</string>
   </property>
  </action>
  <action name="actionSaveRight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить правое изображение</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
